cmd/support-notifications/res/configuration.toml

```diff
@@ -34,12 +34,18 @@ Type = 'consul'
 
 [Smtp]
   Host = 'smtp.gmail.com'
-  Username = 'username@mail.example.com'
-  Password = ''
+  Username = 'username@mail.example.com' # deprecated in V2
+  Password = '' # deprecated in V2
```

Comment:
These need to be removed and replaced with SMTP added to InsecureSecrets section like this:
```
    [Writable.InsecureSecrets.SMTP]
    path = "smtp"
      [Writable.InsecureSecrets.SMTP.Secrets]
      username = "username@mail.example.com"
      password = ""
```

Reply:
we plan to remove them after https://github.com/edgexfoundry/edgex-go/pull/3492 merged, or it would cause V1 code failure

Reply:
Added the SMTP credential for insecure mode.